src/eth/Web3Service.js

```diff
@@ -16,20 +16,20 @@ export default class Web3Service extends PrivateService {
     this._ethersProvider = null;
     this._blockListeners = {};
     this._currentBlock = null;
-    this._info = { version: {} };
+    this._info = {};
     this._statusTimerDelay = TIMER_DEFAULT_DELAY;
     this._defaultEmitter = null;
     this._transactionSettings = null;
 
     Web3ServiceList.push(this);
   }
 
-  version() {
-    return this._info.version;
+  info() {
```

Comment:
~info~ is a more descriptive term and won't cause confusion with web3 0.20.6 documentation